### PR TITLE
Add GraphQL mutation to edit/update profiles

### PR DESCRIPTION
**Pull request by sduenas**
This mutation allows to update profile identities. Some functions were added to the API to update profiles in the database layer.

This commit adds the features requested on #184

### CODE REVIEW

File: sortinghat/core/api.py [hunk 1]
```diff
@@ -201,3 +202,47 @@ def delete_identity(uuid):
...
+    This function allows to edit or update the profile information
+    of the unique identity identified by `uuid`.
+
+    The values to update are given as keyword arguments. The allowed
+    keys are listed below (other keywords will be ignored):
+
+       - 'name' : name of the unique identity
+       - 'email' : email address of the unique identity
+       - 'gender' : gender of the unique identity
+       - 'gender_acc' : gender accuracy (range of 0 to 100; by default,
```
Comment (valeriocos) [inline sortinghat/core/api.py]: According to the tests, this should be:
`gender accuracy (range of 1 to 100; by default, set to 100)`

Comment (sduenas) [reply sortinghat/core/api.py]: Fixed!

Comment (valeriocos) [reply sortinghat/core/api.py]: thanks!

**Comment by valeriocos** [comment]
LGTM

**Pull request merged**